shared/images/iconfont/SKETCH.md

```diff
@@ -0,0 +1,91 @@
+## Sketch Best Practices for SVG Export
+
+The Sketch assets for each icon directly determine the final SVG output that is used to generate the iconfont `kb.ttf`. If there are issues with the colors/curves of the SVG files, then it likely caused by Sketch or the SVGO optimizations done by Zeplin.
+
+### Fill-rule - even-odd v.s. non-zero
+
+When two distincy `path` elements of an SVG overlap, there are two different ways the itersection can be handled using the `fill-rule` property of SVG groups (`g` element).
+
+Reference: [The Winding Order of the Fill Rule](https://oreillymedia.github.io/Using_SVG/extras/ch06-fill-rule.html)
+
+Reference: [MDN fill-rule page](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/fill-rule)
+
+**evenodd**
+
+> The value evenodd determines the "insideness" of a point in the shape by drawing a ray from that point to infinity in any direction and counting the number of path segments from the given shape that the ray crosses. If this number is odd, the point is inside; if even, the point is outside.
+> 
+> Source: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/fill-rule#evenodd
+
+
+**nonzero**
+> The value nonzero determines the "insideness" of a point in the shape by drawing a ray from that point to infinity in any direction and then examining the places where a segment of the shape crosses the ray. Starting with a count of zero, add one each time a path segment crosses the ray from left to right and subtract one each time a path segment crosses the ray from right to left. After counting the crossings, if the result is zero then the point is outside the path. Otherwise, it is inside.
+> 
+> Source: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/fill-rule#nonzero
+
+
+In Sketch it's possible to set the fill-rule for the `icon area` layer which will determine the fill-rule on the final SVG group.
```

Comment:
There's a fill rule on the path layer as well, and setting it to Non Zero seems to help having a clean icon.

Reply:
Ah right on, this is the right answer, thanks for figuring this out the other day!